packages/visual-editor/src/components/migrations/0050_update_link_align_prop.ts

```diff
@@ -0,0 +1,53 @@
+import { Migration } from "../../utils/migrate.ts";
+
+export const updateLinksAlignmentMigration: Migration = {
+  ExpandedFooter: {
+    action: "updated",
+    propTransformation: ({ styles, ...props }) => {
+      const primaryFooter = styles?.primaryFooter;
+      if (!primaryFooter) {
+        return { ...props, styles };
+      }
+
+      if (primaryFooter.linksPosition && !("linksAlignment" in primaryFooter)) {
+        return { ...props, styles };
+      }
+
+      const { linksAlignment, ...restPrimaryFooter } = primaryFooter;
+
+      return {
+        ...props,
+        styles: {
+          ...styles,
+          primaryFooter: {
+            ...restPrimaryFooter,
+            linksPosition:
+              primaryFooter.linksPosition ?? linksAlignment ?? "right",
```

Comment:
I don't think the first `primaryFooter.linksPosition` check is necessary since you have the condition on line 12. The layout shouldn't have both linksPosition and linksAlignment.